frontend/src/types/IntakeAppliedPermit.ts

```diff
@@ -0,0 +1,7 @@
+import type { IStamps } from '@/interfaces';
+
+export type IntakeAppliedPermit = {
+  permitTypeId?: number;
+  statusLastVerified?: string;
+  trackingId?: string;
+} & Partial<IStamps>;
```

Comment:
Is this necessary over the existing `Permit` type, or a partial of? This is just a specific subset of that.

Reply:
good call